<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 255, 203);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="patientButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>病人列表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deviceButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>设备列表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bindButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>病人与设备绑定</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="realtimeButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>实时波形</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="historyButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>历史波形</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>288</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="patientList">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 226, 240);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>查找</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="deviceList">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 224, 255);</string>
      </property>
      <widget class="QTableView" name="deviceView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>431</width>
         <height>371</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>390</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>加入设备</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>420</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>删除设备</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>390</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="bindPage">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 181, 156);</string>
      </property>
      <widget class="QTableView" name="patient_device_tableView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>411</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="patientComboBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>330</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="deviceCombox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>330</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="bindOpButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>330</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>绑定</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>修改所有</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>390</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>解除绑定</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="realtimeWave">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 255, 231);</string>
      </property>
      <widget class="QWidget" name="ecgwid" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>421</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="historyWave">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 255, 220);</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
